src/cow-react/common/pure/RateInfo/index.tsx

```diff
@@ -155,7 +156,13 @@ export function RateInfo({
       )}
       <div>
         <RateWrapper onClick={() => setCurrentIsInversed((state) => !state)}>
-          <span title={currentActiveRate.toFixed(rateOutputCurrency.decimals) + ' ' + rateOutputCurrency.symbol}>
+          <span
+            title={
+              currentActiveRate.toFixed(rateOutputCurrency.decimals || DEFAULT_DECIMALS) +
```

Comment:
Shouldn't this be:
```suggestion
              currentActiveRate.toFixed(rateOutputCurrency.decimals ?? DEFAULT_DECIMALS) +
```

In JavaScript `console.log(0 || 18)` will print 18, but AFAIU, this should be `0`:

```
> 0 || 18
18
> 0 ?? 18
0
```

Reply:
This value is used for presentation, so in this case we want default decimals.

Reply:
Nick is right. If a token has 0 decimals makes no sense to display it with default decimals.

Unless... 
... this is the price representation. 

In that case, it should use the price default decimals, not the one from the currency.